client/galaxy/scripts/apps/analysis.js

```diff
@@ -83,6 +84,7 @@ window.app = function app( options, bootstrapped ){
             '(/)user(/)(:form_id)' : 'show_user_form',
             '(/)workflow(/)' : 'show_workflows',
             '(/)pages(/)(:action_id)' : 'show_pages',
+            '(/)histories(/)(:action_id)' : 'show_histories',
```

Comment:
Edits to comments don't show up in notifications ;)

If it's just two routes, `histories/list` and `histories/published_lists`, both should be defined in buildapp.py as explicit routes instead of catchall actions on a base route.  This is probably the easiest way to ensure they get the correct HTTP response.